### PR TITLE
Increased FW launch abort pitch/roll stick dead band 

### PR DESCRIPTION
**Pull request by breadoven**
Increases fixed wing launch pitch/roll stick abort dead band to reduce likelihood of inadvertent aborts caused by accidentally knocking stick during launch. Dead band required to abort launch increased to a fixed value 250us.

### CODE REVIEW

File: src/main/navigation/navigation_private.h [hunk 1]
```diff
@@ -47,6 +47,8 @@
 
 #define NAV_RTH_TRACKBACK_POINTS            50      // max number RTH trackback points
 
+#define LAUNCH_ABORT_STICK_DEADBAND         250     // pitch/roll stick deflection for lanch abort (us)
```
Comment (JulioCesarMatias) [inline src/main/navigation/navigation_private.h]:
```suggestion
#define LAUNCH_ABORT_STICK_DEADBAND         250     // pitch/roll stick deflection for launch abort (us)
```

Comment (breadoven) [reply src/main/navigation/navigation_private.h]: Fixed

**Comment by DzikuVx** [comment]
cherry picked to release branch

**Comment by Jetrell** [comment]
@breadoven I don't mean to complain... I tested this feature and the stick abort range is at 66% (333uS) either side of mid stick. 
I really like the idea of having the extended deadband, _and I'm fine with it as is_... Although, I did think originally, that it was 125uS either side of center (250uS total).
66% may be a little extreme for some people? They not think to move the stick that far, and assume its not cancelling. 
What do you think about this?

**Comment by Jetrell** [comment]
@breadoven I don't mean to complain... I tested this feature, and the stick abort range can be reasonable greater that 250uS either side of mid stick, due to it falling in the RC Expo range (mine was 330uS, 66%) ... I did think originally, that it was 125uS either side of center (250uS total).
I really like the idea of having the extended deadband, and I'm fine with it as is.
But 250uS + Expo, may be a little extreme for some people? They may not think to move the stick that far, and assume its not cancelling.
Whats your view on this?

**Comment by b14ckyy** [comment]
@Jetrell too much deadband to abourt launch can be very bad. Some planes have insane roll rates or a bad stall tendency. Having to push the stick too far for an actual launch abort and take over can be fatal. I have a setup that has a roll rate of 800dps. Even 250us deadband would cause a commanded 400dps setpoint (not considering expo) instantly when the plane switches from Launch to Acro. This can be fatal when it is just 10m off the ground or so.

**Pull request merged**